pages/What is indentation and why does it matter%3F.md

```diff
@@ -1,13 +1,13 @@
-title:: What is indentation and why does it matter?
+title:: Using Indentation
 
-- Indentation is one of the fundamental concept in Logseq (and outliners in general). It's the feature that allows block references and queries to work. In other words: indentation helps you to find stuff in Logseq.
-- The idea of indentation is simple: anytime you hit the `Tab` key, the block your cursor is in moves to the right. Indenting a block _nests_ it under the block above, creating a parent-child relationship:
+- Indentation is an important visual representation in Logseq (and outliners in general). It allows an view of a hierarchical section of your graph on a page. Representing hierarchies (mind maps, file directories, taxonomies, course prerequisites, org charts, etc) is an important visual tool. Logseq's indentation allows for this type of on-page presentation.
+- Indenting is simple: anytime you hit the `Tab` key, the block your cursor is in moves to the right. Indenting a block _nests_ it under the block above, creating a parent-child relationship (a link):
 	- ![parent-child-example-letters.png](../assets/parent-child-example-letters_1641572097841_0.png)
-- Let's walk through the example step-by-step:
+- Let's walk through the structure:
 	- Block **A** is a parent block; **B** and **C** are _children_ of **A**; block **D** is a _grandchild_ of **A**.
 	- Block **B** is a _child_ of **A** and it's in the same _branch_ as blocks **C** and **D**.
 	- Block **C** is a _child_ of **A** and the _parent_ of **D**.
-- Why is this important? As we've discussed in the previous lessons, Logseq is a networked note-taking tool that works with links. We also discussed that Logseq has no hierarchy, which is actually nuanced. As you've probably deduced by now, hierarchy in Logseq happens at the block level.
+- Why is this important? As we've discussed in the previous lessons, Logseq is a networked note-taking tool that works with links. Again, it's a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)). Graphs can contain hierarchies. A hierarchy is one type of a graph. Indentation allows us to quickly and visually create a small hierarchical section of your graph on a page.
```

Comment:
This is a beginner friendly page and most of our users aren't technical so wikipedia links like this are more confusing than they are helpful